broad-wb-rluckom.json

```diff
@@ -1,10 +1,26 @@
 {
   "name": "terra",
   "owner": "broad-wb-rluckom",
-  "terraform_image": "broadinstitute/terraform:0.11",
+  "terraform_image": "us.gcr.io/broad-dsp-gcr-public/config-render:rl_0",
   "intent": "terra infrastructure",
   "production": false,
   "stability": "volatile",
+  "profile_vars": {
+    "thurloe-configs": {
+      "env": {
+        "GITHUB_TOKEN_VAULT_PATH": "secret/dsp/github/dsdejenkins2/githubtoken",
+        "INSTANCE_TYPE": "",
+        "RUN_CONTEXT": "live",
+        "APP_NAME": "thurloe",
+        "ENVIRONMENT": "qa",
+        "GIT_REPO": "firecloud-develop",
+        "FIRECLOUD_DEVELOP_GIT_BRANCH": "dev",
+        "HOST_TAG": "broad-wb-rluckom-qa",
+        "SERVICE_GIT_SHA_12_CHAR": "4c11a7177674",
+        "TARGET_DOCKER_VERSION": "18.09.6"
+      }
+    }
+  },
```

Comment:
I think a lot of these should be defaults, and not needed to specify here. Things  like `TARGET_DOCKER_VERSION` or `GIT_REPO` won't change very often between files.

Reply:
yeah will do